shap/actions/_optimizer.py

```diff
@@ -9,17 +7,17 @@
 
 
 class ActionOptimizer:
-    def __init__(self, model, actions: list[Action | list[Action]]):
+    def __init__(self, model, actions):
```

Comment:
I assume these changes are since you are not working on the most recent master branch or something got messed up while resolving merge conflicts? Or is there a particular reason to change this?